### ISSUE TITLE
LinuxFixes suggestion: Fix the (move|scroll)-on-button-(down|up) bug

### ISSUE THREAD

**Issue by iamgreaser** [ljoonal/Resonite-Mods#5]
Upstream issue: https://github.com/Neos-Metaverse/NeosPublic/issues/2930

Firstly, thank you so much for fixing the broken rotation and the backwards scroll wheel.

The problem I've mentioned in the linked issue has been a pain for the past... however many months now? It's fixed in more recent releases of the Unity Input library, and if I recall correctly, the backwards scroll wheel is also fixed in more recent releases.

In short, Neos repeats the previous mouse motion OR scroll wheel event whenever a mouse button is pressed or released.

More detail is provided in the upstream issue.

I'm not sure how much work this involves, but I would love to see this bug fixed.

(Side note before anyone asks: I am not able to use Discord. But I can be found in-game as GreaseMonkey.)

**Comment by l-2-j** [comment]
Kind of hard for me to fix as I haven't noticed this behavior. Of course I'm open to implementing a fix, but I have no plans on spending a lot of time investigating it due to not being able to reproduce the issue and check if my changes fixed it quickly locally.
